src/v2/components/AddBlock/components/AddBlockPasteUploader/index.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useState, useCallback } from 'react'
 import styled from 'styled-components'
-import { MutationFn } from 'react-apollo'
+import { MutationFunction as MutationFn } from 'react-apollo'
```

Comment:
Probably already aware but heads up about the new `@apollo/react-hooks` package: 
```js
import { useQuery, useLazyQuery } from "@apollo/react-hooks"
```

Reply:
Ya, that's mostly why I'm upgrading :)